plugins/aea-ledger-ethereum-flashbots/LICENSE

```diff
@@ -186,7 +186,8 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2023 Valory AG
+   Copyright 2021 Valory AG
+   Copyright 2019 Fetch.AI Limited
```

Comment:
I followed the example of the `aea-ledger-ethereum-hwi` plugin which was also developed recently but has this `LICENCE`. 
However, this does not seem to be correct.

Reply:
No it should not have fetchai. Fetchai contributed nothing to either of these plugins

Reply:
Addressed on 3abeea78c5bf87594fc05be1ec2b943b7eedeabc.